components/hdw-bzr/hdw-bzr.c

```diff
@@ -402,28 +409,44 @@ void IRAM_ATTR bzrStopNote(buzzerPlayTrack_t track)
 static bool IRAM_ATTR buzzer_check_next_note_isr(gptimer_handle_t timer, const gptimer_alarm_event_data_t* edata,
                                                  void* user_ctx)
 {
-    // Don't do much if muted
+    // Don't do much if muted or paused
     if ((0 == bgmVolume) && (0 == sfxVolume))
```

Comment:
Should this be checking `bzrPaused` too? Not sure if that's what the comment is saying since there is a check down below.

Reply:
Oh, I was initially checking `bzrPaused` here, but if that happens, `tLastLoopUs` doesn't get updated so when you unpause there's a massive time skip which sounds... not good on the buzzer. I'll fix the comment

Reply:
I thought about this for 30s more and moved the muted check to where the pause check is. No giant time skips for anyone!